packages/shared/src/components/RichTextEditor/plugins/ButtonPlugin/ButtonMarkupElement/ButtonMarkupElementNode.tsx

```diff
@@ -0,0 +1,94 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useRichTextEditorContext } from '@frontify/fondue';
+import { ButtonStyles } from '@frontify/fondue/dist/components/RichTextEditor/Plugins/TextStylePlugin/TextStyles';
```

Comment:
Please fix the import paths, we shouldn’t import from “dist” (Same for other files if any)